bin/tf

```diff
@@ -174,15 +178,22 @@ def update_digest(error: str, path: str, variables: Dict[str, str]) -> bool:
 
     lock_id = '{bucket_name}/{path}.tfstate-md5'.format(
         bucket_name=terraform_bucket,
-        path=re.sub('.*/terraform-config', 'terraform-config', path)
-    )
-    response = dynamodb.upsert_item(
-        table_name='terraform-locking',
-        primary_key_name='LockID',
-        primary_key_value=lock_id,
-        attribute_name='Digest',
-        attribute_value=digest
+        path=calc_repo_path(path=path),
     )
+    if digest:
+        response = dynamodb.upsert_item(
+            table_name='terraform-locking',
+            primary_key_name='LockID',
+            primary_key_value=lock_id,
+            attribute_name='Digest',
+            attribute_value=digest
+        )
+    else:
+        response = dynamodb.delete_item(
```

Comment:
Why do this?

Reply:
You can end up in a bad state if you delete the S3 state file where the new digest value is blank. Dynamodb complained when I tried to set the digest to empty, so deleting the lockid seems like a reasonable next step.